### DISCUSSION TITLE
"Perfect" combo displayed when score isn't an FC

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#15354] [Q&A] [answered]
The game thinks that I got a "perfect" combo when I didn't actually FC. I missed a slider tick on one of the sliders at about halfway through the map and it broke my combo, hence why I only have 737k score instead of having a score closer to 1 million. I set this score in a playlist and I got a 656x combo when an FC score is actually 1052x combo.

Playlist: https://osu.ppy.sh/multiplayer/rooms/114659
Beatmap: https://osu.ppy.sh/beatmapsets/782791#osu/1643641

My score:
![osu_2021-10-29_02-05-15](https://user-images.githubusercontent.com/74088315/139391987-7a25ebb5-e677-4fda-bc3f-563a276c7565.png)

Perfect score:
![osu_2021-10-29_02-13-05](https://user-images.githubusercontent.com/74088315/139391997-e9c1e0b8-1934-4500-99c0-0cf83f0006cf.png)

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/15355

**Comment by TheFantasticLoki** [comment]
Making a comment here as I got a perfect combo on a map where I didn't miss but I did slider break (Difficulty Adjust and Classic Mods used) on the latest release build for windows (https://github.com/ppy/osu/releases/tag/2022.226.0)

Streamable Video showing the sliderbreak in question and the results screen showing a perfect combo: https://streamable.com/hhrf3d

**Reply by frenzibyte** [reply]
This issue has not been resolved yet, and is specifically referring to slider breaks. See https://github.com/ppy/osu/issues/13238.
